README.md

```diff
@@ -78,6 +78,13 @@ In this demonstration, we use a multi-table dataset of 3 million online grocery
 
 For more examples of how to use Featuretools, check out our [demos](https://www.featuretools.com/demos) page.
 
+## Testing
```

Comment:
These instructions should get added to the development section in the install page (https://docs.featuretools.com/getting_started/install.html#development).

You can create a development section in the readme and like to that url too.

Reply:
Ah. I hadn't looked in the docs, and had checked the readme instead. To avoid duplication, would it make sense just to link from the readme to the docs url?

Reply:
get_data_path